classes/local/form/settings_form.php

```diff
@@ -180,11 +180,6 @@ public function define_presignedurl_section($mform) {
         $text = $OUTPUT->heading(get_string('settings:presignedurl:warning', 'tool_objectfs').$link, 6);
         $mform->addElement('html', $OUTPUT->notification($text, 'warning'));
 
-        $mform->addElement('advcheckbox', 'enablepresignedurls',
```

Comment:
this should not have changed, it's the master on off switch regardless of which store is used